mypyc/emitmodule.py

```diff
@@ -104,32 +177,62 @@ def encode_bytes_as_c_string(b: bytes) -> Tuple[str, int]:
     return '"{}"'.format(escaped), len(b)
 
 
-class ModuleGenerator:
+def pointerize(s: str, name: str) -> str:
+    """Given C type and a variable name, return a declaration of a pointer to it."""
```

Comment:
I'm not sure if the docstring matches the implementation. It looks like `s` should have both a type and a variable name, since we are replacing `name` in it. Also, is it possible that there will be multiple matches to replace? I assume that the variable name needs to be sufficiently unique.